### PR TITLE
wip: start writing new index

### PR DESCRIPTION
**Pull request by plt-amy**
I'll be honest, it feels barren and I have no idea what to put there.

### CODE REVIEW

**Comment by Lavenza** [comment]
[Pull request preview](https://preview.1lab.dev/346)

File: support/web/css/code.scss [hunk 1]
```diff
@@ -17,16 +17,20 @@ code, pre, .sourceCode {
...
   font-weight: 500;
+  font-variant-ligatures: none;
 }
 
+code, pre { width: min-content; }
+
 div.sourceCode {
   @include material(var(--code-bg));
 
+  overflow-y: scroll;
```
Comment (ncfavier) [inline support/web/css/code.scss:28]: Why this change? The added scrollbars are kind of ugly.

screenshot-2024-01-19-210552

**Review by ncfavier — COMMENTED** [review]
Looks great! Here's what I think is missing so far:

- a word or two on the foundations we use (homotopy type theory, cubical Agda)
- some sort of broad description of the kind of mathematics we have (e.g. you'll find tons of category theory, not so much complex analysis), maybe give a quick tour of the top-level module namespaces like we had in the old index
- hint at the "all pages" button, which is kind of hidden in the UI

File: support/web/css/code.scss (hunk 1 above)
Comment (ncfavier) [reply support/web/css/code.scss]: Some math blocks are still getting scrollbars:

screenshot-2024-01-19-215542

Comment (plt-amy) [reply support/web/css/code.scss]: okay now *that's* funky

File: src/index.lagda.md [hunk 2]
```diff
@@ -2,1062 +2,162 @@
...
-```
-
-The purpose of the "web of concepts" approach is to let each reader
-approach the 1lab at their own pace: If you already know what all of the
-code above means, you can click on `univalence`{.Agda} to be taken
-directly to the construction of the equivalence --- but if you _don't_,
-you can click on other definitions like `is-equiv`{.Agda} and
-`path→equiv`{.Agda}, and in turn explore the dependencies of _those_
-concepts, and so on.
+Making the formalisation explorable requires taking the Web-native
```
Comment (SquidDev) [inline src/index.lagda.md:28]:
```suggestion
Making the formalisation explorable requires taking the web-native
```

File: src/index.lagda.md [hunk 3]
```diff
@@ -2,1062 +2,162 @@
...
-A formalised, cross-linked reference resource for cubical methods in
-Homotopy Type Theory. Unlike the [@HoTT], the 1lab is not a "linear"
-resource: Concepts are presented as a directed graph, with links
-indicating _dependencies_. For instance, the statement of the univalence
-principle depends on [[universes]], [[identifications|path]] and
-[[equivalences]].  In addition to the hyperlinked "web of concepts"
-provided by the Agda code, there is a short introduction to homotopy
-type theory: **[Start here](1Lab.intro.html)**.
+This website is an experiment in **discoverable formalisation**: an
+extensive library of formalised mathematics, presented as an explorable
```
Comment (SquidDev) [inline src/index.lagda.md:8]: I think it's good that we're being more broad about what the 1Lab formalises, but I do think it's worth mentioning the cubical nature of the formalisation early on, somewhere in the first few sentences?

File: src/index.lagda.md [hunk 4]
```diff
@@ -2,1062 +2,162 @@
...
+- Pages can be looked up by their module name (e.g. _[Algebra.Ring]_),
+  or by their title (e.g. _[Abelian groups]_);
 
-In addition to the _extensive_ use of [[displayed categories]] to model
-"pre-indexing" in the 1Lab, we also contain an in-progress formalisation
-of [Foundations of Relative Category Theory][frct].
+- Sections within a page can be looked up by their title, like _[The
+  canonical self-indexing > As a fibration][selfindex]_. Keep in mind that
+  the fuzzy matching means you don't have to write out the entire
+  prefix; it's only displayed so you know where you'll be taken.
```
Comment (SquidDev) [inline src/index.lagda.md:64]: Everywhere else we refer to "the reader", but here uses "you". We should probably use "the reader" here too.

File: src/index.lagda.md [hunk 5]
```diff
@@ -2,1062 +2,162 @@
...
+[Abelian groups]: Algebra.Group.Ab.html
+[selfindex]: Cat.Displayed.Instances.Slice.html#as-a-fibration
 
-open import Cat.Internal.Morphism
--- Internal monos, epis, and isos.
+While it is infeasible to maintain a list of everything noteworthy, we
+do maintain an association between definitions in the 1Lab and in the
+HoTT book: the [HoTT](HoTT.html) page. It's worth mentioning explicitly
+that this page is also indexed for search, so you can find definitions
+by searching for their name in the book.
```
Comment (SquidDev) [inline src/index.lagda.md:80]:
```suggestion
that this page is also indexed for search, so you can find a definition
by searching for its name in the book.
```

File: src/index.lagda.md [hunk 6]
```diff
@@ -2,1062 +2,162 @@
...
 
-open import Cat.Internal.Reasoning
--- Reasoning combinators for internal categories.
+## Getting involved
 
-open import Cat.Internal.Opposite
--- The opposite of an internal category.
+Even though the 1Lab is maintained primarily by [the authors][authors],
+it is first and foremost a community project. The literal work takes
```
Comment (SquidDev) [inline src/index.lagda.md:85]: Maybe "practical work"? Or "the formalisation"? "literal" is a bit ... messy.

File: src/index.lagda.md [hunk 7]
```diff
@@ -2,1062 +2,162 @@
...
+[LICENSE.EBGaramond]: static/licenses/LICENSE.EBGaramond.txt
+[LICENSE.Noto]: static/licenses/LICENSE.Noto.txt
+[LICENSE.Octicons]: static/licenses/LICENSE.Octicons.txt
+[LICENSE.KaTeX]: static/licenses/LICENSE.KaTeX.txt
+
+[KaTeX]: https://katex.org
+
+## Recently added files
+
+::: {#recent-additions}
```
Comment (SquidDev) [inline src/index.lagda.md:159]: Oooooh, this is snazzy.

File: src/index.lagda.md [hunk 8]
```diff
@@ -2,1062 +2,162 @@
...
-open import Homotopy.Space.Torus -- The torus
-```
+[GitHub]: https://github.com/plt-amy/1lab
+[Discord]: https://discord.gg/eQDNM26uKP
+[authors]: Authors.html
+
+Our purpose is to make univalent mathematics accessible to everyone who
+is interested, regardless of cultural background, age, educational
+background, ethnicity, gender identity, or gender expression. **We
+believe HoTT is for everyone and are committed to fostering a kind,
```
Comment (SquidDev) [inline src/index.lagda.md:97]: "We believe" makes the statement less strong.

```suggestion
background, ethnicity, gender identity, or gender expression. **HoTT
is for everyone and are committed to fostering a kind,
```

**Review by SquidDev — CHANGES_REQUESTED** [review]
Just a couple of phrasing tweaks. Really like what you've done here :heart:.

File: src/index.lagda.md (hunk 4 above)
Comment (ncfavier) [reply src/index.lagda.md:64]: I see more "you"s than "reader"s, and I think "you" is more natural.

Comment (plt-amy) [reply src/index.lagda.md:64]: Both times "reader" shows up it's in the construction _each reader_, though. I don't think there's a change to be made here

**Review by ncfavier — DISMISSED** [review]

**Pull request merged**